README.rst

```diff
@@ -63,13 +65,27 @@ Settings list
 -------------
 All settings are contained in a ``PUSH_NOTIFICATIONS_SETTINGS`` dict.
 
-In order to use GCM, you are required to include ``GCM_API_KEY``.
-For APNS, you are required to include ``APNS_CERTIFICATE``.
+In order to use GCM, you are required to include one of ``GCM_API_KEY``, ``GCM_API_KEYS``, or ``GCM_API_KEYS_MODEL``.
+For APNS, you are required to include ``APNS_CERTIFICATE``, ``APNS_CERTIFICATES``, or ``APNS_CERTIFICATES_MODEL``.
 
 - ``APNS_CERTIFICATE``: Absolute path to your APNS certificate file. Certificates with passphrases are not supported.
+- ``APNS_CERTIFICATES``: A dictionary reflecting separate application IDs to separate APNS certificate files.
+- ``APNS_CERTIFICATES_MODEL``: A dictionary containing description of a database model to reflect
```

Comment:
Fair enough, point made regarding unproven - for that particular library. What if there were a 'more proven' library?

Are you opposing any (optional) database storing of keys/certs completely? Because then we would be on diverging paths.